packages/core/rollup.config.js

```diff
@@ -92,6 +94,17 @@ export default [
       entryFileNames: 'redux-saga-[name].[format].js',
     },
   }),
+  createConfig({
+    input: multiInput,
+    output: {
+      dir: 'dist',
+      format: 'esm',
+      entryFileNames: 'redux-saga-[name].[format].mjs',
```

Comment:
are u aware of the fact that generated chunks (not entry files) contain a hash in their filenames and you will have to handle it? redux-saga update is probably going to change the hash each time

Reply:
I'm also not fond of using .mjs for this, just make it smth like:
```js
entryFileNames: 'redux-saga-[name]. esmodules-browsers.js',
```